packages/editor/src/components/list-view-sidebar/style.scss

```diff
@@ -49,3 +49,8 @@
 	}
 }
 
+.block-editor__tabbed-sidebar {
+	@include break-medium() {
+		width: 350px;
+	}
+}
```

Comment:
This style also defines the inserter sidebar's width, so it feels like it's blurring the lines of the inserter and list view.